### PR TITLE
Things I did to get commcare-android setup/running

### PR DESCRIPTION
**Pull request by millerdev**
@ctsims @amstone326 cc @orangejenny

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -42,11 +42,6 @@ case "`uname`" in
     ;;
 esac
 
-# For Cygwin, ensure paths are in UNIX format before anything is touched.
-if $cygwin ; then
```
Comment (wpride) [inline gradlew]: I think @ctsims wanted to test any changes around the windows functionality

Comment (millerdev) [inline gradlew]: This change was done automatically, I think due to upgrading gradle. I don't know how important it is or if it's even necessary at all. Deferring to more experienced mobile devs on that call.

Comment (wpride) [inline gradlew]: Cool. I think the test is just "Clayton makes sure this branch still builds on his machine"

Comment (wpride) [inline gradlew]: @ctsims think this is just awaiting your sign-off on this issue

**Comment by ctsims** [comment]
@damagatchi retest this please

**Comment by ctsims** [comment]
@millerdev Test failure looks real

**Comment by millerdev** [comment]
@ctsims I'm having trouble understanding the test failure from what I can tell this is the pertinent information from the consolue output in Jenkins:
```
org.commcare.android.tests.processing.FormStorageTest > testAllExternalizablesInPrototypeFactory FAILED
    java.lang.AssertionError at FormStorageTest.java:296

115 tests completed, 1 failed
:app:testCommcareDebugUnitTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:testCommcareDebugUnitTest'.
> There were failing tests. See the report at: file:///home/jenkins/workspace/commcare-android-pr-job-builder/commcare-android/app/build/reports/tests/testCommcareDebugUnitTest/index.html
```
When I look at `FormStorageTest.java:296` I see
```java
assertTrue(
        "Please keep test list up-to-date by adding '" + className + "' to list",
        completeHistoryOfExternalizableClasses.contains(className));
```
Not sure what I did to trigger that assertion failure, and I'm not sure where to find `file:///home/jenkins/workspace/commcare-android-pr-job-builder/commcare-android/app/build/reports/tests/testCommcareDebugUnitTest/index.html`, which maybe has more information about the failure?

Can you either point me to where I can find more detailed failure output or tell me what I'm overlooking?

**Comment by wpride** [comment]
@damagatchi retest this please

**Comment by wpride** [comment]
@millerdev @amstone326 @ctsims Tinkered with this a bit, few points:

1. The first failure was actually a failure on master which was dealt with already, resolved by merging in latest master
2. Build continued failing, speculated this was due to increased strictness around unneeded translations in the updated gradle tools version, so resolved those. But, build continued to fail
3. Finally noticed failure was occurring during the publish test results step, specifically the results files were in the folder `commcare-android/app/build/test-results/testCommcareDebugUnitTest/TEST-*.xml` rather than `commcare-android/app/build/test-results/testCommcareDebugUnitTest/testCommCareDebug/TEST-*.xml` as they had been previously. Changing this path on Jenkins causes build to succeed.

I assume this is just a quirk of upgrading the Gradle version, can't see any other changes in the PR that would cause this.

**Comment by millerdev** [comment]
Thanks for fixing @wpride!

**Pull request merged**